build-utils/src/org/pentaho/wadl/PentahoResourceDoclet.java

```diff
@@ -0,0 +1,129 @@
+package org.pentaho.wadl;
```

Comment:
I would love to see this class be extensible/pluggable with a collection of IDocContributors or something which can be plugged in. This is fine for now though